Description: Upstream changes introduced in version 1.1.0+r236-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 mdaq-roody (1.1.0+r236-1) unstable; urgency=low
 .
   * New pacakge name: mdaq-roody
   * Add new debian package: mdaq-rootana
   * New dependency: mdaq-root 2.3.0+r5196
 .
 The person named in the Author field signed this changelog entry.
Author: Exaos Lee <Exaos.Lee@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- mdaq-roody-1.1.0+r236.orig/rootana/testODB.cxx
+++ mdaq-roody-1.1.0+r236/rootana/testODB.cxx
@@ -12,6 +12,8 @@
 #include <assert.h>
 #include <signal.h>
 
+#include <cstring>
+
 #include "TMidasOnline.h"
 #include "TMidasFile.h"
 #include "TMidasEvent.h"
--- mdaq-roody-1.1.0+r236.orig/rootana/event_dump.cxx
+++ mdaq-roody-1.1.0+r236/rootana/event_dump.cxx
@@ -16,6 +16,9 @@
 #include "TMidasEvent.h"
 #include "TMidasFile.h"
 
+#include <cstring>
+#include <cstdlib>
+
 #ifdef HAVE_ROOT
 #include "XmlOdb.h"
 #endif
--- mdaq-roody-1.1.0+r236.orig/roody/include/DataSourceTNetFolder.h
+++ mdaq-roody-1.1.0+r236/roody/include/DataSourceTNetFolder.h
@@ -97,7 +97,7 @@ class DataSourceTNetFolder: public DataS
     {
       printf("ReadRemoteObject %s from folder %s\n", objname, foldername);
 
-      uint32_t p = 0;
+      uintptr_t p = 0;
       
       {
 	std::string str = "GetPointer ";
